src/index.js

```diff
@@ -344,8 +373,13 @@ function createModificationsHandler(
     const url = await getExperienceUrl(ns.config);
     let res;
     if (url && new URL(url, window.location.origin).pathname !== window.location.pathname) {
-      // eslint-disable-next-line no-await-in-loop
-      res = await replaceInner(new URL(url, window.location.origin).pathname, el);
+      if (metadata?.resolution === 'redirect') {
```

Comment:
I would sanitize it with `toClassName` so we are a bit more flexible on how authors actually write the value:
```suggestion
      if (toClassName(metadata?.resolution) === 'redirect') {
```